packages/account/src/Sections/Profile/FinancialAssessment/financial-assessment.jsx

```diff
@@ -91,7 +92,7 @@ const ConfirmationModal = ({ is_visible, toggleModal, onSubmit }) => (
 
 const ConfirmationPage = ({ toggleModal, onSubmit }) => (
     <div className='account__confirmation-page'>
-        <span className='account__confirmation-page-title'>{localize('Notice')}</span>
+        <Text className='account__confirmation-page-title'>{localize('Notice')}</Text>
```

Comment:
Here, we have replaced text with span but why don't we pass the props? like weight? since, I can see that the className has style like this: 

` &-title {
            color: var(--brand-red-coral);
            height: 20px;
            font-size: 1.4rem;
            font-weight: bold;
            margin-bottom: 0.8rem;
        }`

Reply:
And we can then remove the classname styles except   `margin-bottom: 0.8rem;` because only this text component is using it.

Reply:
Sorry your right, I'm fixing

Reply:
I need to keep color, margin, and height. We don’t support these and color `var(--brand-red-coral)` on text

Reply:
No problem Amir, thanks for changing. 
For colors which are not supported on theme.scss (in the list of colors which starts with --text) you can pass it by `style` prop. We have adde recently. Please merge your PR with master.

Reply:
For example:
`<Text size='s' style={{color: 'var(--brand-red-coral)'}}>`